### PR TITLE
Salesforce source documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Salesforce source documentation

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Singer Salesforce extract source](https://www.notion.so/rudderstacks/Singer-Salesforce-extract-source-f7a7cb57108c441abc8a785fa644b1ee)

### CODE REVIEW

**Comment by ameypv-rudder** [comment]
@chaitanyapraveen the doc is ready for your review: https://deploy-preview-768--rudderstack-docs-staging.netlify.app/sources/extract/salesforce-v2/ You can preview the doc here.

File: docs/sources/extract/salesforce-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+You can further connect this source to your data warehouse by clicking on **Add Destination**, as shown:
+
+<img src="../../assets/cloud-extract-sources/add-destination.png" alt="Adding a destination" />
+
+<div class="successBlock">
+Use the <strong>Use Existing Destination</strong> option if you have an already-configured data warehouse destination in RudderStack. To configure a data warehouse destination from scratch, select the <strong>Create New Destination</strong> button.
+</div>
+
+## Sync dependency on Salesforce API quota
```
[the remark this answers is not in the document]
Comment (am6010) [reply docs/sources/extract/salesforce-v2.mdx:83]: I don't think this is relevant anymore. Based on the description of the [connector](https://docs.airbyte.com/integrations/sources/salesforce/) the synch process will stop if we hit the rate limt.

File: docs/sources/extract/salesforce-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+<img src="../../assets/cloud-extract-sources/salesforce-connection-settings-1.png" alt="Salesforce credentials" />
+
+- Enable the **Sandbox** checkbox if your app is in the Salesforce [sandbox](https://help.salesforce.com/s/articleView?id=sf.create_test_instance.htm&type=5) mode.
+
+<div class="warningBlock">
+It is highly recommended to not check this box if you're unsure whether your app is in a Salesforce sandbox.
+</div>
+
+- Click **Connect with Salesforce V2** and connect your Salesforce account with RudderStack.
+- **Start Date**: Select the start date from when you want RudderStack to ingest the Salesforce data.
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce-v2.mdx:35]:
```suggestion
- **Start Date**: Select the start date from where you want RudderStack to ingest the Salesforce data.
      - If you specify the start date, RudderStack selects all the Salesforce data associated with the `updated` field available from that date and does not replicate any data before this date.
      - If you do not specify the start date, RudderStack replicates all the data associated with the `created` field by default.
      
      
```

File: docs/sources/extract/salesforce-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+<div class="warningBlock">
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce-v2.mdx:37]: can remove this

File: docs/sources/extract/salesforce-v2.mdx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+RudderStack will not replicate any data before this date. Also, RudderStack will replicate all your data by default if you do not set this field.
+</div>
+
+<div class="infoBlock">
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce-v2.mdx:41]: can remove this

File: docs/sources/extract/salesforce-v2.mdx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+<img src="../../assets/cloud-extract-sources/add-destination.png" alt="Adding a destination" />
+
+<div class="successBlock">
+Use the <strong>Use Existing Destination</strong> option if you have an already-configured data warehouse destination in RudderStack. To configure a data warehouse destination from scratch, select the <strong>Create New Destination</strong> button.
+</div>
+
+## FAQ
+
+### Can I connect my Salesforce source to multiple data warehouse destinations?
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce-v2.mdx:85]: Not sure if we should repeat this as we are mentioning it in a note at the top? Also, we are not adding this in every extract doc (acc to the template)

File: docs/sources/extract/salesforce-v2.mdx [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+You can connect <strong>only one data warehouse destination</strong> per Cloud Extract source. If you wish to send data to multiple warehouses, you can configure multiple Cloud Extract sources with the same settings and connect them to each data warehouse.
+
+### How does RudderStack sync my Salesforce data?
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce-v2.mdx:89]: kind of a similar thought, everything here is above in the doc as well

File: docs/sources/extract/salesforce/index.mdx [hunk 7]
```diff
@@ -114,16 +107,17 @@ RudderStack handles these API quotas as described below:
...
 <div class="infoBlock">
-
 Refer to <a href="https://developer.salesforce.com/docs/atlas.en-us.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_api.htm">this</a> article by Salesforce to calculate and increase your total API calls.
 </div>
 
-## FAQs
+## FAQ
 
-#### Is it possible to have multiple Cloud Extract sources writing to the same schema?
+### Can I connect my Salesforce source to multiple data warehouse destinations?
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce/index.mdx:115]: same as above

File: docs/sources/extract/salesforce-v2.mdx (hunk 5 above)
Comment (ameypv-rudder) [reply docs/sources/extract/salesforce-v2.mdx:85]: I know, it was a suggestion from a dev to add this FAQ despite it being repetitive, especially for Salesforce doc.

File: docs/sources/extract/salesforce-v2.mdx [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+<img src="../../assets/cloud-extract-sources/salesforce-connection-settings-1.png" alt="Salesforce credentials" />
+
+- Enable the **Sandbox** checkbox if your app is in the Salesforce [sandbox](https://help.salesforce.com/s/articleView?id=sf.create_test_instance.htm&type=5) mode.
+
+<div class="warningBlock">
+It is highly recommended to keep the checkbox unchecked if you're unsure whether your app is in a Salesforce sandbox.
+</div>
+
+- Click **Connect with Salesforce V2** and connect your Salesforce account with RudderStack.
+- **Start Date**: Select the start date when you want RudderStack to ingest the Salesforce data. RudderStack will not replicate any data before this date.
```
Comment (ishachopra0401) [inline docs/sources/extract/salesforce-v2.mdx:41]:
```suggestion
- **Start Date**: Select the start date from where you want RudderStack to ingest the Salesforce data. RudderStack will not replicate any data before this date.
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**